GestureHandler.web.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import {
+  ScrollView,
+  Slider,
+  Switch,
+  TextInput,
+  ToolbarAndroid,
+  ViewPagerAndroid,
+  DrawerLayoutAndroid,
+  WebView,
+  FlatList,
+  TouchableWithoutFeedback,
+  findNodeHandle,
+} from 'react-native';
+import gestureHandlerRootHOC from './gestureHandlerRootHOC';
+
+const State = {
+  UNDETERMINED: 'UNDETERMINED',
```

Comment:
Can we move it out somewhere?

Reply:
What should we move out in detail? State constants?

Reply:
Yes. If you need them here, I wish to have them in separated file in order not to duplicate code.

Reply:
Sure, makes more than sense 😊 Was not sure before since these constants were used in `GestureHandler.js` as well.

Reply:
Oh, well, we intentionally wanted to take them from native site every time. How I think there's no need to do it. You could think about removing extracting from from Java and Objective-C if it's not big deal for you.

Reply:
Hm, the constants are used in the native part as well. Hence it actually makes sense to read them in the JavaScript part from the native side. Should I undo the changes in `GestureHandler.js` for the time being?

Reply:
No, it's fine now